webapp/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactComponent as IconArrowBack } from './icons/arrow-back.svg';
+import { ReactComponent as IconArrowForward } from './icons/arrow-forward.svg';
+import { ReactComponent as IconChat } from './icons/chat.svg';
+import { ReactComponent as IconEdit } from './icons/edit.svg';
+import { ReactComponent as IconFile } from './icons/file.svg';
+import { ReactComponent as IconLogin } from './icons/login.svg';
+import { ReactComponent as IconLogout } from './icons/logout.svg';
+import { ReactComponent as IconPlay } from './icons/play.svg';
+import { ReactComponent as IconSearch } from './icons/search.svg';
+import { ReactComponent as IconUsers } from './icons/users.svg';
+
+export enum IconType {
+  ARROW_BACK = 'arrowBack',
+  ARROW_FORWARD = 'arrowForward',
+  CHAT = 'chat',
+  EDIT = 'edit',
+  FILE = 'file',
+  LOGIN = 'login',
+  LOGOUT = 'logout',
+  PLAY = 'play',
+  SEARCH = 'search',
+  USERS = 'users',
+}
+
+type SVGPropsType = React.SVGProps<SVGSVGElement> & {
+  title?: string | undefined;
+};
+
+type IconComponentType = React.FunctionComponent<SVGPropsType>;
+
+const Icons: { [key in IconType]: IconComponentType } = {
+  arrowBack: IconArrowBack,
+  arrowForward: IconArrowForward,
+  chat: IconChat,
+  edit: IconEdit,
+  file: IconFile,
+  login: IconLogin,
+  logout: IconLogout,
+  play: IconPlay,
+  search: IconSearch,
+  users: IconUsers,
+};
+
+type IconPropsType = SVGPropsType & {
+  type: IconType;
+};
+
+export function Icon({ type, ...restProps }: IconPropsType) {
```

Comment:
I've been playing around with this and I think importing icons as React components could be problematic. The ideal situation would be if we could expose only props that we want to be configured: (type, size, color etc.) But importing SVG like this forces you to expose all svg properties as props which we don't want... (see the spreading of `restProps` :sweat: )

Maybe there is a workaround but I haven't been able to do it, the ideal would be something like:

```tsx
export function Icon({ type, color, size }: IconProps) {
```

The only way I know to solve this is just to make our custom tsx and inject only desired proporties to SVG html element

Reply:
Good point. I've added the `color` and `size` types to the `IconPropsType` and removed the `SVGPropsType`.
Now we can pass only those props to the SVG component.
What do you think about this option?

Reply:
I've also added the `className` prop to style using CSS classes and `title` for accessibility

Reply:
As discussed on Slack, we will expose only the `type`, `color`, and `size` for now.